### PR TITLE
add image builder

### PR DESCRIPTION
**Pull request by QuentinPerez**

### CODE REVIEW

File: image-builder/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+# Install packages
+RUN apt-get -qq update     \
+ && apt-get -y -qq upgrade \
+ && apt-get install -y -qq \
+      s3cmd                \
+      git                  \
+      curl                 \
+ && apt-get clean
+
+# Download the awesome scaleway-cli ;)
```
Comment (QuentinPerez) [inline image-builder/Dockerfile]: :+1:

File: image-builder/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,29 @@
+## -*- docker-image-name: "armbuild/scw-service-tryit:latest" -*-
```
Comment (moul) [inline image-builder/Dockerfile]: should be `armbuild/scw-image-builder:latest`

File: image-builder/Makefile [hunk 3]
```diff
@@ -0,0 +1,15 @@
+DOCKER_NAMESPACE =	armbuild/
+NAME =			scw-image-builder
+VERSION =		1.0
+VERSION_ALIASES =
```
Comment (moul) [inline image-builder/Makefile]: add `version_aliases = latest`

File: image-builder/Makefile [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+TITLE =			image-builder
+DESCRIPTION =		An image to build others
+SOURCE_URL =		https://github.com/scaleway/image-tools
```
Comment (moul) [inline image-builder/Makefile]: Should be https://github.com/scaleway/image-tools/tree/master/image-builder

File: image-builder/README.md [hunk 5]
```diff
@@ -0,0 +1,13 @@
+Image-builder on Scaleway
+========================
+
+
+Links
+-----
+
+- [Online Labs' Documentation](https://doc.cloud.online.net)
```
Comment (moul) [inline image-builder/README.md]: Service try-it's README.md is outdated, review the README.md file based on image-app-docker instead

File: image-builder/install.sh [hunk 6]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+apt-get install -y s3cmd
+curl -L https://github.com/scaleway/scaleway-cli/releases/download/v1.3.0/scw-Linux-armv7 > /usr/local/bin/scw && chmod +x /usr/local/bin/scw
```
Comment (moul) [inline image-builder/install.sh]: You already downloaded scw in the Dockerfile

File: image-builder/patches/usr/local/bin/image-builder-configure [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+    access = sys.stdin.readline().strip()
+    token = getpass.getpass("Token (hidden): ")
+    with os.fdopen(os.open('/root/.s3cfg', os.O_WRONLY | os.O_CREAT, 0600), 'w') as handle:
+        handle.write(generate_s3cfg_file(access, token))
+    with os.fdopen(os.open('/root/.scwrc', os.O_WRONLY | os.O_CREAT, 0600), 'w') as handle:
+        handle.write(generate_scwrc_file(access, token))
+
+#FIXME: put S3_URL in /etc/bash.bashrc
+
+main()
```
Comment (moul) [inline image-builder/patches/usr/local/bin/image-builder-configure]: AMAZING PYTHON CODE <3<3<3

File: image-builder/Makefile [hunk 8]
```diff
@@ -0,0 +1,15 @@
+DOCKER_NAMESPACE =	armbuild/
+NAME =			scw-image-builder
+VERSION =		1.0
+VERSION_ALIASES =
+TITLE =			image-builder
+DESCRIPTION =		An image to build others
```
Comment (aimxhaisse) [inline image-builder/Makefile]: An image to build other images

**Comment by moul** [comment]
LGTM

**Comment by aimxhaisse** [comment]
LGTM

**Pull request merged**